src/main/java/org/benetech/servicenet/service/impl/DocumentUploadServiceImpl.java

```diff
@@ -114,4 +114,15 @@ public void delete(UUID id) {
         log.debug("Request to delete DocumentUpload : {}", id);
         documentUploadRepository.deleteById(id);
     }
+
+    private String getRealProviderName(String currentProviderName) {
+        if (userService.isAdmin()) {
+            return currentProviderName;
+        }
+
+        //TODO: Replace with provider name connected to the user
+        return userService.getUserWithAuthorities().map(User::getLogin).orElseThrow(
```

Comment:
this piece of code is strange, but I guess it's a todo?

Reply:
Yes, when User will be connected to the System Account, we will return its name here, but for now, it's just login

Reply:
As it's already done, I've updated this PR to cover this.